linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/Emitter.scala

```diff
@@ -462,7 +481,7 @@ final class Emitter(config: Emitter.Config, prePrinter: Emitter.PrePrinter) {
     }
 
     // Static-like methods
-    for (methodDef <- linkedMethods) {
+    for (methodDef <- linkedMethods_!) { // versioning per member inside
```

Comment:
Method set is not tracked? (This looks like a bug which we should be able to reproduce on HEAD).